cli/Cargo.toml

```diff
@@ -82,7 +82,7 @@ serde = { version = "1.0.126", features = ["derive"] }
 shell-escape = "0.1.5"
 sourcemap = "6.0.1"
 swc_bundler = "0.46.0"
-swc_common = { version = "0.11.0", features = ["sourcemap"] }
+swc_common = { version = "0.11.4", features = ["sourcemap"] }
```

Comment:
Is the version bump required for this PR? Normally we bump all the other uses of swc_common across the crates before a release.

Reply:
Yup. Need it for some changes I did in swc_common.